packages/protocol/contracts/governance/Validators.sol

```diff
@@ -1083,4 +1083,25 @@ contract Validators is
     emit ValidatorDeaffiliated(validatorAccount, affiliation);
     return true;
   }
+
+  bytes32[] canForceDeaffiliation = [
```

Comment:
Make this a constant

Reply:
Tried, got
`TypeError: Constants of non-value type not yet implemented.` 
Let me know if you know a way around this